tests/ui/suggestions/issue-89640.stderr

```diff
@@ -0,0 +1,13 @@
+error: expected one of `!`, `.`, `::`, `;`, `?`, `{`, `}`, or an operator, found `t`
+  --> $DIR/issue-89640.rs:2:8
+   |
+LL |     le t x: i32 = 3;
+   |        ^ expected one of 8 possible tokens
+   |
+help: consider removed the space to spell keyword `let`
```

Comment:
```suggestion
help: consider removing the space to spell keyword `let`
```

Sorry, I made a spelling error myself in the initial version of my suggestion. Thanks for applying my feedback! :)